### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by ds5678** [asieradzk/RL_Matrix#7] [General]
Would you consider a more permissive license? Affero GPL is prohibitive for real world use cases and is demotivating for developmental contributions.

**Comment by asieradzk** [comment]
I would consider more permissive license, probably MIT but I have zero motivation to do it right now. You see the project is still very young :) I hope to polish it up & especially the godot plugin & publish a paper solo by the time I finish my PhD - so you see I am not too thrilled about receiving PRs that much. I am however happy to discuss academic collaboration.

**Comment by asieradzk** [comment]
I've decided to change to dual license, MIT for personal/academic use but I am still not keen on accepting contributions. If people are interested I'm going to draft CLA and have them sign it.
